taranis/analyze_schema.py

```diff
@@ -140,37 +221,65 @@ def fetch_statistics_from_alleles(self, a_quality):
         record_data["good_percent"] = round(
             100 * (total_alleles - bad_quality_counter) / total_alleles, 2
         )
-        for item in possible_bad_quality:
-            record_data[item] =  bad_quality_reason[item] if item in bad_quality_reason else 0
-        # record_data["bad_quality_reason"] = bad_quality_reason
+        for item in taranis.utils.POSIBLE_BAD_QUALITY:
```

Comment:
I think you can do a double list comprenhension here, but not sure

Reply:
Yes, I will modify it with this code
```
        labels = taranis.utils.POSIBLE_BAD_QUALITY
        values = [stats_df[item].sum() for item in labels]
```

Reply:
it is implemented this solution in code